### PR TITLE
quiet aPTP daemon

### PR DESCRIPTION
**Pull request by mrakes**
- lower the debug level of a couple of VERY verbose prints, and comment out a couple of hard-coded
    prints that are just too much to have in the regular system log

(another change is forthcoming that will move aPTP logging from a file in /var/log into the standard /var/log/messages stream)

### CODE REVIEW

**Comment by sailorsf** [comment]
the commit for runtime log level is https://github.com/bowerswilkins/OpenAvnu/commit/8d6b5e636a50ef27adbc2dc40c849cf61fbe1a83

**Comment by mrakes** [comment]
@sailorsf if you look a few lines above the std::cout... that same info is already debug printed. Just think we got somebody's "this works, ship it" version of the daemon code with this release. :)

That said, I will pull in your debugging changes, though verbose is so crazily verbose I can't see why anybody would ever want to use it :)

File: daemons/gptp/common/gptp_log.cpp [hunk 1]
```diff
@@ -40,6 +40,8 @@ std::mutex gLogKeeper;
 DLT_DECLARE_CONTEXT(dlt_con_gptp);
 #endif
 
+GPTP_LOG_LEVEL gptp_debug_log_level = GPTP_LOG_LVL_CRITICAL;
```
Comment (mrakes) [inline daemons/gptp/common/gptp_log.cpp]: @sailorsf should this be lower than CRITICAL by default? seems like we should at least set to ERROR?

Comment (sailorsf) [reply daemons/gptp/common/gptp_log.cpp]: agree.

Comment (sailorsf) [reply daemons/gptp/common/gptp_log.cpp]: But as for now, the log level is only checked when printing debug/verbose. all the other log functions do not care about this level yet. We can fix them later if deemed useful. Over all the aPTP code is pretty rough to me. I am expecting more work needs to be done. BTW the verbose can be very useful in debugging... anyhow,

Comment (sailorsf) [reply daemons/gptp/common/gptp_log.cpp]: You may also add filter for all other log levels if you want to suppress them too. but can do it later.

Comment (mrakes) [reply daemons/gptp/common/gptp_log.cpp]: right... yep. I realized this this AM once I thought about it a bit more :)

**Pull request merged**